README.md

```diff
@@ -10,7 +10,7 @@
 
 ### Contributing
 
-OpenEMR is a leader in healthcare open source software and comprises a large and diverse community of software developers, medical providers and educators with a very healthy mix of both volunteers and professionals.  [Join us and learn how to start contributing today!](https://open-emr.org/wiki/index.php/FAQ#How_do_I_begin_to_volunteer_for_the_OpenEMR_project.3F)
+OpenEMR is a leader in healthcare open source softwares and comprises of a large and diverse community of software developers, medical providers and educators with an effective blend of both our volunteers and professionals.  [Join us and learn how to start contributing today!](https://open-emr.org/wiki/index.php/FAQ#How_do_I_begin_to_volunteer_for_the_OpenEMR_project.3F)
```

Comment:
hi @aprakash07 , would leave it as `software and comprises a...`

how about `an effective blend of volunteers and professionals.`?

Reply:
"Softwares" is not a word, as "software" already refers to a _collection_ of programs, instructions etc.

"Comprise" is like "contain" or "include", so we don't say "comprises of".

"Both" and "our" seem superfluous here. I'd just say "an effective blend of volunteers and professionals".

Reply:
yes, an effective blend of volunteers and professionals looks fine.